### PR TITLE
Use a more useful example for rest property destructuring

### PR DESCRIPTION
**Pull request by geryogam**

### CODE REVIEW

File: files/en-us/web/javascript/reference/operators/destructuring_assignment/index.md [hunk 1]
```diff
@@ -268,9 +268,8 @@ You can also ignore all returned values:
 The rest property of array destructuring assignment can be another array or object binding pattern. This allows you to simultaneously unpack the properties and indices of arrays.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/operators/destructuring_assignment/index.md:268]: While testing this, I realized one thing: this syntax accesses properties on the array created _after_ the rest property, not the original iterable. Therefore it's much less useful than I thought. What about:

```suggestion
The rest property of array destructuring assignment can be another array or object binding pattern. The inner destructuring destructures from the array created after collecting the rest elements, so you cannot access any properties present on the original iterable in this way.
```

Comment (geryogam) [reply files/en-us/web/javascript/reference/operators/destructuring_assignment/index.md]: Exactly, I also realized this and it was not obvious so it is worth mentioning. Thanks for this very useful suggestion. Committed.

**Review by Josh-Cena — APPROVED** [review]
Thank you, great improvement!

**Comment by geryogam** [comment]
Thanks @Josh-Cena!

**Pull request merged**